compiler/src/main/scala/org/scalajs/nscplugin/JSPrimitives.scala

```diff
@@ -55,7 +55,7 @@ abstract class JSPrimitives {
   final val CREATE_INNER_JS_CLASS = CONSTRUCTOROF + 1                  // runtime.createInnerJSClass
   final val CREATE_LOCAL_JS_CLASS = CREATE_INNER_JS_CLASS + 1          // runtime.createLocalJSClass
   final val WITH_CONTEXTUAL_JS_CLASS_VALUE = CREATE_LOCAL_JS_CLASS + 1 // runtime.withContextualJSClassValue
-  final val LINKING_INFO = WITH_CONTEXTUAL_JS_CLASS_VALUE + 1          // runtime.linkingInfo
+  final val LINKING_INFO = WITH_CONTEXTUAL_JS_CLASS_VALUE + 1          // runtime.linkingInfo (deprecated since 1.17.0)
   final val IDENTITY_HASH_CODE = LINKING_INFO + 1                      // runtime.identityHashCode
```

Comment:
This is not a public API, so you can completely remove it:
```suggestion
  final val IDENTITY_HASH_CODE = WITH_CONTEXTUAL_JS_CLASS_VALUE + 1    // runtime.identityHashCode
```